code/datums/station_traits/innate_traits.dm

```diff
@@ -0,0 +1,88 @@
+//This file is for innate station traits, which are applied from the map!
+
+#define TREK_VARIATION_TOS "tos"
+#define TREK_VARIATION_TNG "next"
+#define TREK_VARIATION_VOY "voy"
+#define TREK_VARIATION_ENT "ent"
+
+/datum/station_trait/trek
+	name = ""
```

Comment:
needs a name!

Reply:
Just named em "North Star Uniforms", will be a part of the next commit I make!